flow/table.go

```diff
@@ -158,7 +158,8 @@ func NewTable(updateEvery, expireAfter time.Duration, sender Sender, uuids UUIDs
 	return t
 }
 
-func (ft *Table) getFlows(query *filters.SearchQuery) *FlowSet {
+//GetFlows returns flowset
+func (ft *Table) GetFlows(query *filters.SearchQuery) *FlowSet {
```

Comment:
This has to remain private has we can call it from outside because of mutex/lock. This has to call be the flow table through a chan. If we had the option in order to specify whether we want partial updates it should remove the need of thi s modification.